rogue-thi-app/pages/food.js

```diff
@@ -104,222 +125,323 @@ export default function Mensa () {
-                    <br />
-                    {meal.restaurant}
-                  </div>
-                </ListGroup.Item>
-              )}
-            </ListGroup>
-          )}
-          {foodEntries && foodEntries.length === 0 &&
-            <ListGroup>
-              <ListGroup.Item>
-                Der Speiseplan ist leer.
-              </ListGroup.Item>
-            </ListGroup>
-          }
-        </ReactPlaceholder>
-
-        <br />
-
-        <Modal show={showMealDetails} onHide={() => setShowMealDetails(null)}>
-          <Modal.Header closeButton>
-            <Modal.Title>Erläuterung</Modal.Title>
-          </Modal.Header>
-
-          <Modal.Body>
-            <h5>Anmerkungen</h5>
-            {showMealDetails?.flags === null && 'Unbekannt.'}
-            {showMealDetails?.flags?.length === 0 && 'Keine.'}
-            <ul>
-              {showMealDetails?.flags?.map(flag => (
-                <li key={flag}>
-                  <strong>{flag}</strong>
-                  {' – '}
-                  {flagMap[flag] || FALLBACK_ALLERGEN}
-                </li>
-              ))}
-            </ul>
-
-            <h5>Allergene</h5>
-            {showMealDetails?.allergens === null && 'Unbekannt.'}
-            {showMealDetails?.allergens?.length === 0 && 'Keine.'}
-            <ul>
-              {showMealDetails?.allergens?.map(key => (
-                <li key={key} style={{ color: containsSelectedAllergen([key]) && COLOR_WARN }}>
-                  {containsSelectedAllergen([key]) && (
-                    <span>
+                              ))}
+                            </small>
+                          </div>
+                        </div>
+                        <div className={styles.right}>
+                          {getUserSpecificPrice(meal)}
+                          <br />
+                          {meal.restaurant}
+                        </div>
+                      </ListGroup.Item>
+                  )}
+                </ListGroup>
+            )}
+            {foodEntries && foodEntries.length === 0 &&
+                <ListGroup>
+                  <ListGroup.Item>
+                    Der Speiseplan ist leer.
+                  </ListGroup.Item>
+                </ListGroup>
+            }
+          </ReactPlaceholder>
+
+          <br />
+
+          <Modal show={showMealDetails} onHide={() => setShowMealDetails(null)}>
+            <Modal.Header closeButton>
+              <Modal.Title>Erläuterung</Modal.Title>
+            </Modal.Header>
+
+            <Modal.Body>
+              <h5>Anmerkungen</h5>
+              {showMealDetails?.flags === null && 'Unbekannt.'}
+              {showMealDetails?.flags?.length === 0 && 'Keine.'}
+              <ul>
+                {showMealDetails?.flags?.map(flag => (
+                    <li key={flag}>
+                      <strong>{flag}</strong>
+                      {' – '}
+                      {flagMap[flag] || FALLBACK_ALLERGEN}
```

Comment:
Here we could insert the handshake icon as well as green color, when the flag is one of the preferred ones. Similar to how the allergenes turn red (see line 234-238)

Reply:
Also coming with the next push